lib/src/material_tree/group/material_tree_group.dart

```diff
@@ -125,8 +157,49 @@ class MaterialTreeGroupComponent extends MaterialTreeNode implements OnDestroy {
     e.stopPropagation();
   }
 
+  void _sliceOptionGroup(int end) {
+    if (end < group.length) {
+      _visibleGroup = group.slicedOptionGroup(0, end);
+    } else {
+      _visibleGroup = group;
+    }
+  }
+
+  /// Sets option group and visible option group if [maxInitialOptionsShown] is
+  /// specified.
+  @Input()
   @override
-  ngOnDestroy() {
+  set group(OptionGroup _group) {
+    super.group = _group;
+
+    if (_maxInitialOptionsShown == null) {
+      _visibleGroup = group;
+      return;
+    }
+
+    _sliceOptionGroup(_maxInitialOptionsShown);
+  }
+
+  /// Toggles on the collapsed options.
+  void viewMoreOptions(Event event) {
+    event.preventDefault();
+    event.stopImmediatePropagation();
+
+    _visibleGroup = group;
+  }
+
+  /// Whether "View more" link should be visible.
+  bool get viewMoreLinkVisible =>
+      _maxInitialOptionsShown != null && !identical(_visibleGroup, group);
+
+  @override
+  void ngOnDestroy() {
     onDestroy();
   }
+
+  final viewMoreMsg = _viewMoreMsg;
+
+  static String get _viewMoreMsg => Intl.message('View more',
+      desc: 'Label for a link that allows user to see the collapsed options.'
+          '[REL_NOTE: jiahaowang/LAUNCHING-2018-03-31]');
```

Comment:
Was the intent really to end the `desc` string with `'[REL_NOTE: jiahaowang/LAUNCHING-2018-03-31]'`?

Reply:
Thanks, I'll follow up with the author of this change to see what was going on here.

Reply:
Removed.